io.github.DamnedAngel.msx-tile-forge.yaml

```diff
@@ -0,0 +1,39 @@
+app-id: io.github.DamnedAngel.msx-tile-forge
+runtime: org.freedesktop.Platform
+runtime-version: "25.08"
+sdk: org.freedesktop.Sdk
+
+command: msxtileforge
+
+finish-args:
+  - --share=ipc # needed for X11
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=xdg-documents
+
+modules:
+  - name: msx-tile-forge
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/DamnedAngel/msx-tile-forge/releases/download/v01.00.00_rc20_5/msxtileforge_01.00.00_rc20_5_lin.tar.gz
```

Comment:
What happen on aarch64?

Reply:
why does this bundle x86_64 only python, libstdc++, libX11, etc?

It should not.